Experimental/src/org/sleuthkit/autopsy/experimental/configuration/MultiUserTestTool.java

```diff
@@ -153,16 +154,16 @@ static String runTest(String rootOutputDirectory) {
             }
 
             // Make a text file in Windows TEMP folder 
-            String tempFilePath = System.getProperty("java.io.tmpdir") + TEST_FILE_NAME + "_" + TimeStampUtils.createTimeStamp() + ".txt";
+            Path tempFilePath = Paths.get(System.getProperty("java.io.tmpdir"), TEST_FILE_NAME + "_" + TimeStampUtils.createTimeStamp() + ".txt");
```

Comment:
You might want to update the comment since the code is no longer Windows specific.

Reply:
Done ;)